ocp_resources/virtual_machine_snapshot.py

```diff
@@ -77,3 +77,41 @@ def wait_ready_to_use(self, status=True, timeout=TIMEOUT_4MINUTES):
         for sample in samples:
             if sample:
                 return
+
+    def wait_snapshot_done(self, timeout=TIMEOUT_4MINUTES):
+        """
+        Wait for the the snapshot to be done. This check 2 parameters, the snapshot status to be readyToUse
+        and the VM status snapshotInProgress to bu null.
+
+        Args:
+            timeout (int): Time to wait.
+
+        Raises:
+            TimeoutExpiredError: If timeout reached.
+        """
+        self.wait_ready_to_use(timeout=timeout)
+
+        vm_snapshot_status = "snapshotInProgress"
+        vm = list(
```

Comment:
duplicate code; please create a function and pass vm status to it

Reply:
done